packages/rune-games-cli/src/lib/validateGameFiles.ts

```diff
@@ -113,35 +113,41 @@ export async function validateGameFiles(
           logicJs,
           indexHtml,
         })
-      }
-
-      if (scripts.indexOf(sdkScript) !== 0) {
-        errors.push({
-          message: "Rune SDK must be the first script in index.html",
-        })
-      }
-
-      const sdkVersion = sdkScript
-        .getAttribute("src")
-        ?.match(sdkVersionRegex)?.[1]
 
-      if (!sdkVersion) {
-        errors.push({ message: `Rune SDK must specify a version` })
-      }
+        if (scripts.indexOf(sdkScript) !== 0) {
+          errors.push({
+            message: "Rune SDK must be the first script in index.html",
+          })
+        }
 
-      const [major, minor, patch] = (sdkVersion ?? "").split(".")
-      const maxedOutSdkVersion = `${major}.${minor ?? 999}.${patch ?? 999}`
+        const sdkVersion = sdkScript
+          .getAttribute("src")
+          ?.match(sdkVersionRegex)?.[1]
 
-      const sdkVersionCoerced = semver.coerce(sdkVersion && maxedOutSdkVersion)
-      const minSdkVersionCoerced = semver.coerce(minSdkVersion)
+        if (!sdkVersion) {
+          errors.push({ message: `Rune SDK must specify a version` })
+        }
 
-      if (
-        sdkVersionCoerced &&
-        minSdkVersionCoerced &&
-        semver.lt(sdkVersionCoerced, minSdkVersionCoerced)
-      ) {
+        const [major, minor, patch] = (sdkVersion ?? "").split(".")
+        const maxedOutSdkVersion = `${major}.${minor ?? 999}.${patch ?? 999}`
+
+        const sdkVersionCoerced = semver.coerce(
+          sdkVersion && maxedOutSdkVersion
+        )
+        const minSdkVersionCoerced = semver.coerce(minSdkVersion)
+
+        if (
+          sdkVersionCoerced &&
+          minSdkVersionCoerced &&
+          semver.lt(sdkVersionCoerced, minSdkVersionCoerced)
+        ) {
+          errors.push({
+            message: `Rune SDK is below minimum version (included ${sdkVersion}, min ${minSdkVersion})`,
+          })
+        }
+      } else {
         errors.push({
-          message: `Rune SDK is below minimum version (included ${sdkVersion}, min ${minSdkVersion})`,
+          message: `Rune SDK must be multiplayer`,
```

Comment:
Perhaps `Singleplayer Rune SDK is no longer supported`?

Reply:
In this PR I exclusively check if game is multiplayer.
So if dev includes any other file in `index.html`, e.g:
```html
<script src="https://cdn.jsdelivr.net/npm/rune-games-sdk@4.8.1/dist/libs.js"></script>
```
then showing `Singleplayer Rune SDK is no longer supported` would be confusing.

Reply:
Then maybe let's say smth like `Rune SDK script url should end in /multiplayer.js`

Reply:
Sure, I will add `Rune SDK script url must end with /multiplayer.js or /multiplayer-dev.js`

Reply:
Done in https://github.com/rune/rune-games-sdk/pull/223/commits/e3db7655c8d9d55ae56ff37c61b9946ba5eb14b6